modules/imgproc/include/opencv2/imgproc/hal/hal.hpp

```diff
@@ -192,6 +192,12 @@ CV_EXPORTS void cvtTwoPlaneYUVtoBGR(const uchar * src_data, size_t src_step,
                                     int dst_width, int dst_height,
                                     int dcn, bool swapBlue, int uIdx);
 
+//! Separate Y and UV planes
+CV_EXPORTS void cvtTwoPlaneYUVtoBGR(const uchar * y_data, const uchar * uv_data, size_t src_step,
```

Comment:
In IPP they use convention like cvtYUVtoBGR_P2. Probably, we should use it as well.